dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -66,7 +66,7 @@ Future<void> _testBuildIosFramework(Directory projectDir, { bool isModule = fals
   String content = pubspec.readAsStringSync();
   content = content.replaceFirst(
     '\ndependencies:\n',
-    '\ndependencies:\n  device_info: 0.4.1\n  package_info: 0.4.0+9\n  connectivity: 3.0.3\n',
+    '\ndependencies:\n  device_info: 2.0.3\n  package_info: 2.0.2\n  connectivity: 3.0.6\n',
```

Comment:
Grr
```
  "reason": "FileSystemException: Expected file to exist., path = '/opt/s/w/ir/x/t/flutter_module_test.GlHziT/hello_module/flutter-frameworks/Debug/device_info.xcframework/ios-arm64_armv7/device_info.framework/Headers/DeviceInfoPlugin.h'"
```
Working on a fix.

Reply:
And it passed: https://ci.chromium.org/p/flutter/builders/try/Mac%20build_ios_framework_module_test/13177